start

```diff
@@ -433,9 +433,14 @@ function upgrade_standalone() {
 }
 
 function exec_supervisor() {
-  echo "starting supervisor"
+  echo "starting supervisor..."
   upgrade_standalone &
-  exec supervisord -n -c $SUPHOME/etc/supervisord.conf
+  if [ "$ENABLE_SOROBAN_RPC" = "true" ]; then
+      sorobanRpcStatus &
+  else 
+      horizonStatus &   
+  fi    
+  exec supervisord -n -c $SUPHOME/etc/supervisord.conf > /dev/null 2>&1
```

Comment:
removed the supervisord logging output from reaching quickstart console, it was kind of verbose and repetitive, maybe not informative/diagnostic for most end users, by removing, the console output from quickstart seems a bit leaner.

Reply:
yes, I took a brief step back first and a tee approach, so, all logs go to `/quickstart.log` file and then a select amount of those logs that are most pertinent for user viewing are routed to console output, and I included the supervisor output in the console output, here's what the current console output looks like afterwards:

![Screen Shot 2023-01-20 at 3 03 54 PM](https://user-images.githubusercontent.com/10703845/213820867-2081a3d2-9a27-4362-9f1c-406a620741c7.png)

of errors on console, user can shell in and view the full log trace in `/quickstart.log`